Makefile

```diff
@@ -81,15 +84,18 @@ servedocs: docs ## compile the docs watching for changes
 	watchmedo shell-command -p '*.rst' -c '$(MAKE) -C docs html' -R -D .
 
 release: dist ## package and upload a release
-	twine upload dist/*
+	twine upload $(DIST_DIR)/*
```

Comment:
since we use `maturin publish` in the travis file, I think it would be best to just use that here as well

Reply:
I kept this around because it gives us a little more control over exactly what gets uploaded - e.g. maturin doesn't handle just-sdist yet, so decoupling the build and upload steps might be preferable. That issue is raised with maturin, though.

Reply:
Ah ok, thats fine then, I thought the functionality for the two tools was essentially the same